src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -276,15 +271,13 @@ const formConfig = {
               'Your contact information',
               'We use this information to contact you if we have more questions.',
             ),
-            ...noPhoneInfo.uiSchema,
```

Comment:
`noPhoneInfo` was actually just empty JSX (the body of the component was commented out because we thought the copy might change, but we ended up just nixing it). So, these references to it can be deleted.